### DISCUSSION TITLE
2007/01/26/%E8%AE%B0%E5%BD%95%E6%A2%A6%E5%A2%83-5

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#143] [Announcements]
# 2007/01/26/%E8%AE%B0%E5%BD%95%E6%A2%A6%E5%A2%83-5

公司要搬家了，所以昨晚梦到了新的办公室。

https://blog.qizhen.xyz/2007/01/26/%E8%AE%B0%E5%BD%95%E6%A2%A6%E5%A2%83-5

**Comment by ruanqizhen** [comment]
之前博客下的留言：


P
2007年01月29日 20:52 编辑
HELLO，谢谢你来踩我的博！：）

回复
洋
2007年01月28日 15:27 编辑
我的记忆中好象只来过这里几次，留言也是甚少，没有想到还是有人把我加了连接，HOHO~激动一下，我觉得我们都是同一个专业的哦~不然为什么都要学LABVIEW，今年我开始用LABVIEW 8了，发觉真的是这个东西真的是越来越强大~~

回复
Jing
2007年01月27日 12:07 编辑
哈哈哈，想起了金庸小说中的那个梦中砌墙的…………

回复
